Dockerfile.desktop

```diff
@@ -2,13 +2,13 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-FROM ubuntu:mantic as moz-central
+FROM python:3-slim as moz-central
```

Comment:
Q: Do we have stats on what this gives us? 30% image size? or just easier to work with? Or just keeps us on an always recent version of Ubuntu vs pinning to 23.n's Mantic Minotaur?

Reply:
Mainly easier to work with but yes it will be a bit smaller, I am not sure of the exact difference